spec/requests/api/v1/posters_request_spec.rb

```diff
@@ -132,11 +132,12 @@
     original_name = poster.name
     new_name = test_params[:name]
 
-    patch "/api/vi/posters/#{poster.id}", params: test_params.to_json, headers: { "Content-Type" => "application/json" }
-
+    patch "/api/v1/posters/#{poster.id}", params: test_params.to_json, headers: { "Content-Type" => "application/json" }
+    new_poster = JSON.parse(response.body, symbolize_names: true)
+#require 'pry'; binding.pry
     expect(response).to be_successful
-    expect(poster.name).to_eq(new_name)
-    expect(poster.name).not_to equal(original_name)
+    expect(new_poster[:name]).to eq(new_name)
```

Comment:
update to use bracket notation here, this allows us to properly access the `:name` value from a JSON object.